apps/sms/test/marionette/lib/messages.js

```diff
@@ -51,43 +42,17 @@ var ConversationAccessor = require('./views/conversation/accessors');
 
         Composer: new ComposerAccessor(client),
```

Comment:
Mmm, not sure I get the question? :) It's just temporary thing to not rewrite all old tests and it will be removed as soon as view move to "views" entirely (because of the fact that we don't wont expose accessors publicly, only views)...